CMakeLists.txt

```diff
@@ -345,7 +345,11 @@ Enable this if the module uses typeid or dynamic_cast. You will probably need to
             include(sdk/cmake/baseaddress_msvc.cmake)
         endif()
     else()
-        include(sdk/cmake/baseaddress.cmake)
+        if (ARCH STREQUAL "amd64")
+            include(sdk/cmake/baseaddress_gcc_x64.cmake)
+        else()
+            include(sdk/cmake/baseaddress.cmake)
```

Comment:
While here, maybe rename this file to `baseaddress_gcc.cmake`, to be more explicit and match MSVC?